packages/terra-time-input/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "terra-time-input",
+  "main": "lib/TimeInput.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-time-input component is an input for entering time. During time entry the value is masked to conform to the time format.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-time-input",
+    "TimeInput",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "moment": "^2.18.1",
+    "react-text-mask": "^4.1.0",
+    "terra-mixins": "^1.0.0"
```

Comment:
There is a pull request out now that add terra-base as an explicit dependency, you might check out this PR for more information: https://github.com/cerner/terra-core/pull/251